x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/40_semantic_text_query.yml

```diff
@@ -142,6 +174,51 @@ setup:
   - match: { hits.hits.0._id: "doc_1" }
   - length: { hits.hits.0._source.inference_field.inference.chunks: 1 }
 
+---
+"Query using a sparse embedding model via a search inference ID":
+  - requires:
+      cluster_features: "gte_v8.16.0"
```

Comment:
Will this test break serverless? Should we use capabilities here or define a specific cluster feature instead?